openssh/test/controls/service_running.rb

```diff
@@ -0,0 +1,14 @@
+control 'ensure sshd processes are all running and owned by correct owner' do
+
+  describe "openssh processes" do
+    it "/sbin/sshd process running" do
+      expect(command("ps aux | grep -v grep | grep '\/sbin\/sshd' | wc -l").stdout.strip.to_i).to eq(1)
```

Comment:
I believe `ps` and maybe other programs we make use of here are not available within the studio - that might be worth mentioning when explaining the busybox pkg install in the README

Reply:
I've added explanation to the README regarding requirements for the binlinked core/busybox